sacred/utils.py

```diff
@@ -315,6 +315,40 @@ def __init__(
         super().__init__(message, print_traceback, filter_traceback, print_usage)
 
 
+class FilteredTracebackException(tb.TracebackException):
+    def __init__(
+        self,
+        exc_type,
+        exc_value,
+        exc_traceback,
+        *,
+        limit=None,
+        lookup_lines=True,
+        capture_locals=False,
+        _seen=None,
+    ):
+        super().__init__(
+            exc_type,
+            exc_value,
+            exc_traceback,
+            limit=limit,
+            lookup_lines=lookup_lines,
+            capture_locals=capture_locals,
+            _seen=_seen,
+        )
+
+        self.stack = tb.StackSummary.extract(
+            (
+                (tb_frame, tb_lineno)
+                for tb_frame, tb_lineno in tb.walk_tb(exc_traceback)
+                if not _is_sacred_frame(tb_frame)
+            ),
+            limit=limit,
+            lookup_lines=lookup_lines,
+            capture_locals=capture_locals,
+        )
```

Comment:
This, unfortunately, doesn't work for chained exceptions. In that case, the `__init__` of `TracebackException` creates more instances of `TracebackException` that are not filtered.

Reply:
K, maybe we can filter `exc_traceback` bevor the TracebackException class. The type for `exc_traceback` is `<class 'traceback'>.

Reply:
`<class 'traceback'>` is some where in the C code, so I made my own class, which seems to work.